flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/strategy/InputConsumableDecider.java

```diff
@@ -33,14 +33,14 @@ public interface InputConsumableDecider {
      * Determining weather the input of an execution vertex is consumable.
      *
      * @param executionVertexID identifier of this {@link ExecutionVertex}.
-     * @param verticesToDeploy all vertices to deploy during the current scheduling process. This
-     *     set will be used to know whether an execution vertex has been decided to scheduled.
+     * @param verticesToSchedule vertices that are not yet scheduled by already decided to be
+     *     scheduled.
      * @param consumableStatusCache a cache for {@link ConsumedPartitionGroup} consumable status.
-     *     This will be used to reduce double computing.
+     *     This is to avoid repetitive computation.
      */
     boolean isInputConsumable(
             ExecutionVertexID executionVertexID,
-            Set<ExecutionVertexID> verticesToDeploy,
+            Set<ExecutionVertexID> verticesToSchedule,
```

Comment:
This naming issue stays not only in this interface, but also in its implementation and the calculation logics in the strategy.

Reply:
Ah, I forgot to change it.